docs/docs/behaviors/bluetooth.md

```diff
@@ -38,14 +38,15 @@ This will allow you to reference the actions defined in this header such as `BT_
-| `BT_PRV`     | Switch to the previous profile, cycling through to the last one when the beginning is reached.                                                                                     |
-| `BT_SEL`     | Select the 0-indexed profile by number; must include a number as an argument in the keymap to work correctly, e.g. `BT_SEL 0`.                                                     |
-| `BT_DISC`    | Disconnect from the 0-indexed profile by number, if it's currently connected and inactive; must include a number as an argument in the keymap to work correctly, e.g. `BT_DISC 0`. |
+| Define       | Action                                                                                                                                                                                                                                           |
+| ------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `BT_CLR`     | Clear bond information between the keyboard and host for the selected profile.                                                                                                                                                                   |
+| `BT_CLR_ALL` | Clear bond information between the keyboard and host for all profiles.                                                                                                                                                                           |
+| `BT_NXT`     | Switch to the next profile, cycling through to the first one when the end is reached.                                                                                                                                                            |
+| `BT_PRV`     | Switch to the previous profile, cycling through to the last one when the beginning is reached.                                                                                                                                                   |
+| `BT_SEL`     | Select the 0-indexed profile by number; must include a number as an argument in the keymap to work correctly, e.g. `BT_SEL 0`.                                                                                                                   |
+| `BT_DISC`    | Disconnect from the 0-indexed profile by number, if it's currently connected and inactive; must include a number as an argument in the keymap to work correctly, e.g. `BT_DISC 0`.                                                               |
+| `BT_SAVE`    | Saves the currently selected profile, ensuring it is active at startup. <br/><br/>Note: This is only useful if `ZMK_CONFIG_BLE_PERSIST_PROFILE_ON_CHANGE` is disabled. Per default the active profile will be persisted on change automatically. |
 
 :::note[Selected profile persistence]
```

Comment:
Actually it would be good to add a qualifier to this note box as well

Reply:
Added info to this note box about the config as well as `BT_SAVE`.

Reply:
Thanks, the update looks good.